src/components/class-management/class-management.tsx

```diff
@@ -0,0 +1,89 @@
+import { useContext, useEffect, useState } from "react";
+import { Button, Form, Input, message, Table } from "antd";
+import axios from "axios";
+import { AuthContext } from "../../state/reducer";
+import "./class-management.scss";
+
+interface IClass {
+  internalName: string;
+  displayedName: string;
+}
+
+function ClassManagement() {
+  const [classes, setClasses] = useState<IClass[]>([]);
+  const [loading, setLoading] = useState(false);
+  const { token } = useContext(AuthContext);
+
+  useEffect(() => {
+    setLoading(true);
+    axios
+      .get("/classes")
```

Comment:
```suggestion
    axios
      .get<Responses.IGetCLasses("/classes")
```
Add a type for this response in `responses.ts`